hastexo/hastexo.py

```diff
@@ -273,6 +284,7 @@ def launch_or_resume_user_stack(self, sync = False):
         else:
             result = task.apply_async(args=args, kwargs=kwargs, expires=60)
             self.user_stack_launch_id = result.id
+            self.user_stack_launch_timestamp = datetime.datetime.now(pytz.utc)
```

Comment:
Minor note here (not for now, but to be revisited later): I find it a little weird that a timestamp would need to know its applicable timezone.